internal/telemetry/telemetry_test.go

```diff
@@ -151,7 +151,6 @@ func TestRegisterAppConfig(t *testing.T) {
 	require.Equal(t, RequestTypeAppStarted, req.RequestType)
 	appStarted := req.Payload.(*AppStarted)
 	cfg := appStarted.Configuration
-	require.Len(t, cfg, 2)
```

Comment:
@Julio-Guerra Is this change required?

Reply:
yes, because I changed the global config with `dependencies_available` and `DD_TELEMETRY_DEPENDENCY_COLLECTION_ENABLED` reported by default all the times, so I removed the enforcement to 2 and let the following assertions fail anyway if the expected config values are missing.